src/app/locale-selector/locales.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Inject, Injectable, InjectionToken, LOCALE_ID } from "@angular/core";
+import { CookieService } from "ngx-cookie-service";
+export const LOCALES_LIST = new InjectionToken<Array<string>>('List of locales available');
+
+@Injectable()
+export class LocalesService {
+  setLocaleCookie(value: string) {
+    this.cookie.set('bb-locale', value);
```

Comment:
There is the only place, where we use the method from external library and according to the source code it can be changed to `this.document.cookie =  encodeURIComponent(name) + '=' + encodeURIComponent(value) + ';';`.
Do you think it is worth to have it?

Reply:
Maybe we can use local storage?

Reply:
it can be worthy but maybe in the future if we see there are more cookies management. But for now I followed your suggestion and removed the cookie service and made it as you said

Reply:
@kirillsud this should use the cookies because nginx expects that `bb-locale` is present to route to the right locale